extensions/wikia/TemplateDraft/TemplateDraftHooks.class.php

```diff
@@ -2,11 +2,22 @@
 
 class TemplateDraftHooks {
 
+	/*
+	 * property used in page_props table, construction:
+	 * * tc- - prefix for "template classifiaction"
+	 * * -marked- - signifies classification decision made by human
+	 * * -auto- - signifies classification decision made by AI
+	 * * -infobox - suffix denoting the type of template we identified
+	 *
+	 * @TODO move this someplace sensible, once available
+	 */
+	const TEMPLATE_INFOBOX_PROP = 'tc-marked-infobox';
+
 	public static function onGetRailModuleList( Array &$railModuleList ) {
 		global $wgTitle;
 
-		// TODO: Check page props here
-		if ( $wgTitle->getNamespace() === NS_TEMPLATE ) {
+		if ( $wgTitle->getNamespace() === NS_TEMPLATE
+			&& Wikia::getProps( $wgTitle->getId(), self::TEMPLATE_INFOBOX_PROP ) !== 0 ) {
```

Comment:
Coding conventions are to move the closing parenthesis and curly bracket onto the next line when doing multi-line conditionals to make a clearer separation of the conditional from the body, i.e.
```php
if ( $wgTitle->getNamespace() === NS_TEMPLATE
	&& Wikia::getProps( $wgTitle->getId(), self::TEMPLATE_INFOBOX_PROP ) !== 0
) {
	$railModuleList[1502] = [ 'TemplateDraftModule', Index, null ];
}
```